### PR TITLE
Fix Rename Bug, Add Additional File Tags

### LINKED ISSUE #1: Renaming uses script path rather than target directory
Notice from this output the error when renaming the files:

C:\Users\Simon>movie-renamer F:\private\rtorrent\Send\movies
Listing files...
Scraping IMDB results...
?   The.Bourne.Legacy.2012.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
?   The.Bourne.Legacy.2012.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
 -> The Bourne Legacy (2012).mkv: Yes
?   The.Bourne.Ultimatum.2007.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
?   The.Bourne.Ultimatum.2007.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
 -> The Bourne Ultimatum (2007).mkv: Yes
?   The.Bourne.Supremacy.2004.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
?   The.Bourne.Supremacy.2004.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv
 -> The Bourne Supremacy (2004).mkv: Yes
Renaming files...
Done!
Error while renaming "The.Bourne.Legacy.2012.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv": Error: ENOENT: no such file or directory, rename 'C:\Users\Simon\The.Bourne.Legacy.2012.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv' -> 'C:\Users\Simon\The Bourne Legacy (2012).mkv'
Error while renaming "The.Bourne.Ultimatum.2007.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv": Error: ENOENT: no such file or directory, rename 'C:\Users\Simon\The.Bourne.Ultimatum.2007.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv' -> 'C:\Users\Simon\The Bourne Ultimatum (2007).mkv'
Error while renaming "The.Bourne.Supremacy.2004.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv": Error: ENOENT: no such file or directory, rename 'C:\Users\Simon\The.Bourne.Supremacy.2004.2160p.BluRay.REMUX.HEVC.DTS-X.7.1-FGT.mkv' -> 'C:\Users\Simon\The Bourne Supremacy (2004).mkv'

### PR DESCRIPTION
**Pull request by Cyber5imon**
Fixed Rename Bug using Script Path
Added New Tags: 2160p, Atmos, BluRay

Closes #1

### CODE REVIEW

File: renamer.js [hunk 1]
```diff
@@ -163,7 +169,7 @@ function renameFiles(files) {
 
   _.each(files, function(file) {
     if (file.new) {
-      fs.rename(file.original, file.new, function(err) {
+      fs.rename(process.argv[2] + '/' + file.original, process.argv[2] + '/' + file.new, function(err) {
```
Comment (sinedied) [inline renamer.js]: Please use `path.join()` for path concatenation so that the script remains portable (needs also a `path = require('path')` at the top)

Comment (sinedied) [inline renamer.js]: Please use `path.join()` for path concatenation so that the script remains portable (needs also a `path = require('path')` at the top)

**Review by sinedied — CHANGES_REQUESTED** [review]
Thanks for the PR!

I made a small comment to keep the code portable, could you look at it?

After it's fixed, I'll merge that once I get home 😉

**Comment by Cyber5imon** [comment]
Updated fs.rename to use path.join()

**Review by sinedied — APPROVED** [review]
Perfect, thanks!

**Pull request merged**